server_test.go

```diff
@@ -299,50 +309,6 @@ func (l *fakeListener) Accept() (net.Conn, error) {
 	return nil, errors.New("connection closed")
 }
 
-// Test that a connection is closed upon reaching an idle state iff the server is shutting down.
-func TestCloseOnIdle(t *testing.T) {
```

Comment:
Because it is a very very unlikely case that does not really solve anything accept adding one more special case in implementation.